### PR TITLE
New dynamic thema subjects

### PR DESCRIPTION
**Pull request by arzola**
Related Issue #2105 

This PR depends on `https://github.com/pressbooks/symbionts-thema` to download the new thema subjects for each supported language

Thema subjects json file will be downloaded on demand when the book language is updated if there is no thema file for the selected language an en fallback will be loaded instead.

1. Thema files were deleted from the plugin only english version is loaded by default to provide a safe fallback.

### How to test

1. Change the language of your book and review if the subject themas are being loaded.
2. Try to select a unsupported thema language to see if the default english are loaded.

### CODE REVIEW

**Comment by richard015ar** [comment]
Oscar, it LGTM. I tested locally and works as expected.
As we discussed if in the future we will need more actions for thema subjects, probably we could move those actions to a new class if makes sense.

@SteelWagstaff only pending thing is validate the message displayed when a thema subject is not found:
Screenshot from 2021-03-23 14-15-31

File: inc/metadata/namespace.php [hunk 1]
```diff
@@ -1161,3 +1187,66 @@ function get_in_catalog_option() {
...
+		if ( ! function_exists( 'download_url' ) ) {
+			require_once( ABSPATH . 'wp-admin/includes/file.php' );
+		}
+
+		$download_file = "{$thema_lang_baseurl}{$thema_lang}.json";
+
+		$downloaded = download_url( $download_file );
+		if ( is_wp_error( $downloaded ) ) {
+			$_SESSION['pb_errors'][] = sprintf(
+				__( 'Your %1$s thema subjects could not be downloaded from %2$s.', 'pressbooks' ),
```
Comment (SteelWagstaff) [inline inc/metadata/namespace.php:1226]: Replace with 'The %1$s Thema subject terms requested for this book could not be downloaded from %2$s. Please report this error to Pressbooks development team using our [bug report form in GitHub](https://github.com/pressbooks/pressbooks/issues/new?assignees=&labels=&template=bug.md&title=)'

**Comment by richard015ar** [comment]
LGTM!

**Review by richard015ar — APPROVED** [review]
LGTM! thank you Oscar!

### Test notes
- I selected multiple thema objects for a particular book and:
  - Select different languages available for Thema and it works as expected, the subject language changes with the language selected.
  - I selected languages unsupported by Thema and the English version of the subjects are displayed as expected

For test the compatibility with previous versions, I checkout to the previous version of the repo and stored a subject in Japanese (unsupported by previous version of Thema). And then I moved to this branch and the subject was transalated using the new Thema subjects as expected.

**Pull request merged**